quickstarts/jbpm-workitems-camel/pom.xml

```diff
@@ -199,18 +256,97 @@
+                        </manifestEntries>
+                    </archive>
+                    <rebuildBundle>true</rebuildBundle>                    
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.wildfly.plugins</groupId>
+                <artifactId>wildfly-maven-plugin</artifactId>
+                <version>${version.wildfly.maven}</version>
+                <executions>
+                    <execution>
+                        <id>deploy</id>
+                        <phase>install</phase>
+                        <goals>
+                            <goal>deploy-only</goal>
+                        </goals>
+                        <configuration>
+                            <filename>${project.build.finalName}.war</filename>
+                            <skip>${deploy.skip}</skip>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>undeploy</id>
+                        <phase>clean</phase>
+                        <goals>
+                            <goal>undeploy</goal>
+                        </goals>
+                        <configuration>
+                            <filename>${project.build.finalName}.war</filename>
+                            <skip>${deploy.skip}</skip>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <configuration>
+                    <executable>java</executable>
+                    <classpathScope>test</classpathScope>
+                    <mainClass>org.jboss.integration.fuse.quickstarts.jbpm.workitems.camel.ApplicationMain</mainClass>
                 </configuration>
             </plugin>
         </plugins>
     </build>
+  <profiles>
+    <profile>
+      <id>deploy</id>
+      <properties>
+        <deploy.skip>false</deploy.skip>
+      </properties>
+    </profile>
+    <profile>
+      <id>eap</id>
+      <build>
+      	<plugins>
+      	   <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <configuration>
+                    <executable>java</executable>
+                    <classpathScope>test</classpathScope>
+                    <mainClass>org.jboss.integration.fuse.quickstarts.jbpm.workitems.camel.ApplicationMain</mainClass>
+                     <systemProperties>
+                       <systemProperty>
+                         <key>DEFAULT_PORT</key>
+                         <value>8080</value>
+                       </systemProperty>
+                     </systemProperties>
+                </configuration>
+            </plugin>
+      	</plugins>
+      </build>
+    </profile>
+  </profiles>
```

Comment:
Could you please run a code formatter on all the XML files? They all should use 4 spaces as indent level.